web3.js/test/connection.test.js

```diff
@@ -1279,6 +1279,69 @@ test('get supply', async () => {
   expect(supply.nonCirculatingAccounts.length).toBeGreaterThan(0);
 });
 
+test('get performance samples', async () => {
+  if (!mockRpcEnabled) {
```

Comment:
I personally prefer live tests over the mock because they serve as nice integration tests for the rpc api as well as smoke tests for breaking changes that require a web3 fix. Is there something about perf sampling that makes it hard to add a live test?

Reply:
I'll check into this again. At the moment the perf samples API wasn't present during the live test.